KMD.Identity.TestApplications.SAML.MVCCore/Infrastructure/Saml/CustomSamlCertificateValidator.cs

```diff
@@ -0,0 +1,66 @@
+﻿using ITfoxtec.Identity.Saml2.Util;
+using Microsoft.IdentityModel.Tokens;
+using System.Security.Cryptography.X509Certificates;
+using System;
+using System.Linq;
+
+namespace KMD.Identity.TestApplications.SAML.MVCCore.Infrastructure.Saml
+{
+    /// <summary>
+    /// This is an extension to ItFoxtec certificate validator, which will fail when building chain in specific situations:
```

Comment:
This line confused me. At first I read it as this class failing in certain conditions, but as a read the rest of the PR I came the the conclusion that "which" refers to the base class. I think this is caused by the comma which makes the sentence incorrect. However, I suggest rephrasing this to something like the following for extra clarity:

"This is an extension to the ItFoxtec certificate validator which allows you to soften the validation requirements for certificates. The following scenarios will cause certificate validation to fail in the base class but can be ignored in this class using the validation flag:"